review.js

```diff
@@ -372,50 +349,39 @@
       const href = a.getAttribute('href')
       if (!href) continue
       const login = href.slice(1).toLowerCase()
-      const body = comment.querySelector('.comment-body')
-      if (body && LGTM_RE.test(body.innerHTML)) {
+      const paragraphs = comment.querySelectorAll('.comment-body > p')
+      if (Array.from(paragraphs).some(p => LGTM_RE.test(p.innerHTML))) {
```

Comment:
In https://github.com/nodejs/node/pull/10657, a LGTM embedded in a `<blockquote>` (and therefore one that does not belong to the author of the comment) was recognized. This piece of code only looks for direct `<p>` children of `.comment-body`, and mitigates that issue.